Describe plasma_calculator in narrative documentation (#1390)
PlasmaPy/PlasmaPy: docs/plasma_calculator/index.rst

.. _plasmapy-calculator:

=================
Plasma Calculator
=================

.. currentmodule:: plasmapy.utils.calculator

Overview
--------

> Plasma calculator is an interactive notebook installed alongside plasmapy that

Reviewer: ```suggestion
Plasma calculator is an interactive notebook installed alongside PlasmaPy that
```

Author: Done!

allows users to input a set of plasma properties and get immediate output for
various functions at once.

Using Plasma Calculator
-----------------------

> To invoke the app use `plasma_calculator`, by default opens in a browser bootstrapping
> the notebook in a light theme and uses port 8866 by default.

Reviewer: ```suggestion
To invoke the app run ``plasma-calculator`` in the command line. By default 
this opens the app in a browser, bootstrapping the notebook in a light theme 
and using port 8866 by default.
```

> `plasma-calculator` takes optional arguments such as `dark`, `port`, `no-browser` pass flag
> `-h` or `--help` to get full list of supported arguments.

Reviewer: ```suggestion
``plasma-calculator`` takes optional arguments such as ``--dark``, ``--port``,
and ``--no-browser``.  Pass flag ``-h`` or ``--help`` to get the full list of 
supported arguments.
```
These flags need a `--` beforehand, I think?  I also added double back ticks so that the text inside gets interpreted as an in-line code snippet, since single back ticks in reST are used to link to a particular code object.

> To invoke the app use ``plasma_calculator``, by default opens in a browser bootstrapping
> the notebook in a light theme and uses port 8866 by default.
> 
> ``plasma-calculator`` takes optional arguments such as ``--dark``, ``--port``, ``--no-browser`` pass flag
> ``-h`` or ``--help`` to get full list of supported arguments.

Reviewer: ```suggestion
To invoke the app use ``plasma_calculator`` in the command line. 
By default the opens the app in a browser, bootstrapping
the notebook in a light theme and uses port 8866 by default.

``plasma-calculator`` takes optional arguments such as ``--dark``, ``--port``, ``--no-browser`` pass flag
``-h`` or ``--help`` to get full list of supported arguments.
```

.. _plasmapy-calculator:

=================
Plasma Calculator
=================

.. currentmodule:: plasmapy.utils.calculator

Overview
--------

Plasma calculator is an interactive notebook installed alongside PlasmaPy that
allows users to input a set of plasma properties and get immediate output for
various functions at once.

Using Plasma Calculator
-----------------------

> To invoke the app use ``plasma_calculator`` in the command line. 

Reviewer: ```suggestion
To invoke the app use ``plasma-calculator`` in the command line. 
```
